### PR TITLE
Update Oxford-Journals.xml

### PR DESCRIPTION
**Pull request by Folant**

### CODE REVIEW

File: src/chrome/content/rules/Oxford-Journals.xml [hunk 1]
```diff
@@ -1,22 +1,813 @@
...
+	<target host="ywcct.oxfordjournals.org" />
+	<target host="m.ywcct.oxfordjournals.org" />
+	<target host="ywes.oxfordjournals.org" />
+	<target host="m.ywes.oxfordjournals.org" />
 
-	<rule from="^http://(access|secure|services)\.oxfordjournals\.org/"
-		to="https://$1.oxfordjournals.org/" />
+	<securecookie host=".+" name=".+" />
 
+	<rule from="^http://secure\.oxfordjournals\.org/" to="https://oxfordjournals.org/" />
```
Comment (J0WI) [inline src/chrome/content/rules/Oxford-Journals.xml]: Why is this necessary? The server side redirect seems to work fine.

Comment (Folant) [inline src/chrome/content/rules/Oxford-Journals.xml]: Maybe it was broken back then, i don't remember. Removed it.
